devmgmtui/src/components/cloud/CloudDownload.js

```diff
@@ -176,6 +176,7 @@ class CloudDownload extends Component {
 				console.log(err);
 				
 				alert('Trouble fetching content from Diksha server.');
```

Comment:
This is not plugin code, you should not add "Diksha" in the alert message. Change it.

Reply:
The cloud endpoints are Diksha specific, thus the message. This is one of the many places where plugin specific code exists. Thought it would be best to decouple this when I am taking up that issue.